tests/Functional/src/com/microsoft/aad/adal/test/OauthTests.java

```diff
@@ -62,6 +62,7 @@
 import com.microsoft.aad.adal.PromptBehavior;
 
 import android.annotation.SuppressLint;
+import android.test.AndroidTestCase;
```

Comment:
nope, i cannot. I'm seeing whole bunch of error shown in eclipse when reverting it.